src/shared/components/annotation/CancerHotspots.tsx

```diff
@@ -48,15 +48,22 @@ export default class CancerHotspots extends React.Component<ICancerHotspotsProps
         );
     }
 
-    public static title(isHotspot:boolean, is3dHotspot:boolean)
+    public static title(isHotspot:boolean, is3dHotspot:boolean, count?: number, customInfo?: JSX.Element)
     {
-        const recurrentHotspot = isHotspot ? (<b>Recurrent Hotspot</b>) : "";
-        const maybeAnd = isHotspot && is3dHotspot ? "and" : "";
-        const clusteredHotspot = is3dHotspot ? (<b>3D Clustered Hotspot</b>) : "";
+        const recurrentHotspot = isHotspot ? (<b>Recurrent Hotspot</b>) : undefined;
```

Comment:
i think maybe null would better since that's what we use elsewhere to rendering "nothing".  no big deal.